### ISSUE TITLE
Thinq1 Washer Broken since 1.2.6

### ISSUE THREAD

**Issue by ekobres** [nVuln/homebridge-lg-thinq#98]
Plugin stuck constantly trying to setup washer after failing an API call. The Last version of this plugin that works for me is version 1.2.5.

I am seeing the following errors (debug mode) in 1.2.9 - followed by the accessories disappearing from HomeKit:

```
[02/11/2021, 17:13:23] [homebridge-lg-thinq] retried request error:  NotConnectedError: 장치가 접속중이 아닙니다.
    at client.interceptors.response.use.err.response._a (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/request.ts:27:13)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at API.request (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:57:12)
    at API.postRequest (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:46:12)
    at API.thinq1PostRequest (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:252:12)
    at API.sendMonitorCommand (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:176:12)
    at ThinQ.registerWorkId (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/ThinQ.ts:96:31)
    at ThinQ.setup (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/ThinQ.ts:73:7)
    at LGThinQHomebridgePlatform.discoverDevices (/usr/local/lib/node_modules/homebridge-lg-thinq/src/platform.ts:120:28)
[02/11/2021, 17:13:23] [homebridge-lg-thinq] TypeError: Cannot read property 'lgedmRoot' of undefined
    at /usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:254:35
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at API.thinq1PostRequest (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:252:12)
    at API.sendMonitorCommand (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/API.ts:176:12)
    at ThinQ.registerWorkId (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/ThinQ.ts:96:31)
    at ThinQ.setup (/usr/local/lib/node_modules/homebridge-lg-thinq/src/lib/ThinQ.ts:73:7)
    at LGThinQHomebridgePlatform.discoverDevices (/usr/local/lib/node_modules/homebridge-lg-thinq/src/platform.ts:120:28)
[02/11/2021, 17:13:23] [homebridge-lg-thinq] Cannot read property 'lgedmRoot' of undefined
```
I installed latest 1.2.10 test and instead get a Cannot read of a different property than lgedmRoot.

I reverted to 1.2.5 to get my Washer working again.

**Comment by nVuln** [comment]
test version `v1.2.10-test.1` still got error `Cannot read property 'lgedmRoot' of undefined` ? can you upload error log for this ?

**Comment by nVuln** [comment]
I have published test version `v1.2.10-test.2` to fix thinq1 device issue, can you help me install and test it ?

**Comment by ekobres** [comment]
This test version appears to load and start polling, which is definitely much better than recent builds. I will reply back after checking operation of accessories later. Thank you!

**Comment by ekobres** [comment]
Both washer and dryer appear to be polling properly so far. Have not had time yet to see whether occupancy sensor is working. Will close once it fires.

**Comment by nVuln** [comment]
I already released `v1.2.10` to fix this issue, please update, if you got any error please open new issue
